test/algorithms/physical_design/post_layout_optimization.cpp

```diff
@@ -14,48 +14,87 @@
 #include <fiction/layouts/clocked_layout.hpp>
 #include <fiction/layouts/coordinates.hpp>
 #include <fiction/layouts/gate_level_layout.hpp>
+#include <fiction/layouts/obstruction_layout.hpp>
 #include <fiction/layouts/tile_based_layout.hpp>
 #include <fiction/networks/technology_network.hpp>
+#include <fiction/utils/routing_utils.hpp>
 
 #include <mockturtle/networks/aig.hpp>
 #include <mockturtle/views/names_view.hpp>
 
-#include <tuple>
+#include <cstdint>
 #include <vector>
 
 using namespace fiction;
 
 template <typename Lyt, typename Ntk>
-void check_mapping_equiv(const Ntk& ntk)
+void check_layout_equiv(const Ntk& ntk)
 {
     const auto layout = orthogonal<Lyt>(ntk, {});
 
     post_layout_optimization_stats stats{};
-    post_layout_optimization<Lyt>(layout, &stats);
+    post_layout_optimization<Lyt>(layout, {}, &stats);
 
     check_eq(ntk, layout);
 
     CHECK(mockturtle::to_seconds(stats.time_total) > 0);
 }
 
 template <typename Lyt>
-void check_mapping_equiv_all()
+void check_layout_equiv_all()
 {
-    check_mapping_equiv<Lyt>(blueprints::maj1_network<mockturtle::aig_network>());
-    check_mapping_equiv<Lyt>(blueprints::maj4_network<mockturtle::aig_network>());
-    check_mapping_equiv<Lyt>(blueprints::unbalanced_and_inv_network<mockturtle::aig_network>());
-    check_mapping_equiv<Lyt>(blueprints::and_or_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::nary_operation_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::constant_gate_input_maj_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::half_adder_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::full_adder_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::mux21_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::se_coloring_corner_case_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::fanout_substitution_corner_case_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::inverter_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::clpl<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::one_to_five_path_difference_network<technology_network>());
-    check_mapping_equiv<Lyt>(blueprints::nand_xnor_network<technology_network>());
+    SECTION("maj1_network")
+    {
+        check_layout_equiv<Lyt>(blueprints::maj1_network<mockturtle::aig_network>());
+    }
+    SECTION("unbalanced_and_inv_network")
+    {
+        check_layout_equiv<Lyt>(blueprints::unbalanced_and_inv_network<mockturtle::aig_network>());
+    }
+    SECTION("and_or_network")
+    {
+        check_layout_equiv<Lyt>(blueprints::and_or_network<technology_network>());
+    }
+    SECTION("nary_operation_network")
+    {
+        check_layout_equiv<Lyt>(blueprints::nary_operation_network<technology_network>());
+    }
```

Comment:
These `SECTION`s contain some superfluous newlines

Reply:
Sorry I'm not sure what you mean, would you prefer:
```SECTION("nary_operation_network")
    {check_layout_equiv<Lyt>(blueprints::nary_operation_network<technology_network>());}```

Reply:
Sorry, it was a rendering issue on my side. All is well.